test/libsolidity/semanticTests/viaYul/array.sol

```diff
@@ -0,0 +1,21 @@
+contract C {
+	uint[] storageArray;
+	function test_storage_array() public returns (uint256)
```

Comment:
If you really want to test this, then you'd have to test *clearing* of storage... which is of course hard if we can't really *write* to storage or *read* from storage at this time without inline assembly :-)... well, maybe it's fine for actual tests be postponed for yet another PR later...

Reply:
I implemented them in the index PR already